cypress-shared/support/common/testcase.js

```diff
@@ -202,48 +202,6 @@ export function startE2E(app, workspace) {
   })
 }
 
-function verifyEnvs(paymentEnvs = false) {
-  if (apiKey && apiToken) {
-    if (paymentEnvs) {
-      if (appKey && appToken && affiliationId) {
-        cy.log('All envs are available in VTEX_QE.json')
-      } else {
-        cy.log('Some envs are missing in VTEX_QE.json')
-      }
-    }
-  } else {
-    cy.log('Some envs are missing in VTEX_QE.json')
-  }
-}
-
-export function startPaymentE2ETests() {
-  // If you are using this function in your testcase
-  // then ensure, you are having affiliationId, apiKey and apiToken in .VTEX_QE.json
-  it(`Start ${prefix} E2E tests with this workspace ${WORKSPACE}`, () => {
-    verifyEnvs(true)
-    cy.request({
```

Comment:
instead of dropping completely have this function
let this function call verifyEnvs() and verify whether we have required envs in vtex_qe.json or not

Reply:
ok